submissions/webdevtoday/oop_exercise/index.js

```diff
@@ -0,0 +1,80 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Mammal {
+   constructor(name, gender, saying, species, friends) {
+      this.name = name;
+      this.gender = gender;
+      this.species = species;
+      this.saying = saying;
+      this.friends = friends;
+      this.legs = 4;
+   }
+   getProperties() {
+      return [this.species, this.name, this.gender, this.legs, this.friends.join(', ')];
+   }
+   [Symbol.toPrimitive](hint) {
+      if (hint == 'string') {  
+         return this.getProperties().join(';');
+      }
+   }
+}
+
+class Human extends Mammal {
+   constructor(name, gender, saying, friends) {
+      super(name, gender, saying, 'human', friends);
+      this.legs = 2;
+      this.hands = 2;
```

Comment:
Adding `hands` as an own property of a `Human` is absolutely required as it is a property unique to humans.

`legs` is a shared property. If didn't have any assumptions as to the default value for it in `Mammal` you would manage this in a better way just like `name`, `species` etc.

Ask yourself, why 4 is a default value for `legs`? Are there any good reasons to have a default value for it in `Mammal`? Other properties do not deserve to have default values in `Mammal` so what's different with `legs`?